### ISSUE TITLE
console.log('Node')

### ISSUE THREAD

**Issue by Bubujka** [krg7880/json-schema-generator#18]
Is it intended to always print "Node" in the bin/cli.js ?
https://github.com/krg7880/json-schema-generator/blob/master/bin/cli.js#L3

With this line its harder to use json-schema-generator in pipes: 
I need to put "| tail -n +2" for receiving correct json document 

`cat package.json  | json-schema-generator | tail -n +2 | json2yml`
Instead of
`cat package.json  | json-schema-generator | json2yml`

**Comment by mo** [comment]
Yes, very annoying. Looks like @krg7880 accidentally added it in a commit called called "Fixed test suite"
